src/schema/v2/partner/partnerInquirerCollectorProfile.ts

```diff
@@ -1,80 +1,19 @@
 import {
-  GraphQLBoolean,
   GraphQLFieldConfig,
   GraphQLFieldConfigMap,
-  GraphQLID,
-  GraphQLList,
-  GraphQLNonNull,
   GraphQLObjectType,
-  GraphQLString,
 } from "graphql"
 import { ResolverContext } from "types/graphql"
-import date from "schema/v2/fields/date"
-import Image, { normalizeImageData } from "schema/v2/image"
 import { CollectorProfileFields } from "schema/v2/CollectorProfile/collectorProfile"
-import { myLocationType } from "schema/v2/me/myLocation"
-
-export const CollectorProfileArtists = new GraphQLObjectType<
-  any,
-  ResolverContext
->({
-  name: "CollectorProfileArtists",
-  fields: {
-    name: { type: GraphQLString },
-  },
-})
 
 const InquirerCollectorProfileFields: GraphQLFieldConfigMap<
   any,
   ResolverContext
 > = {
   ...CollectorProfileFields,
```

Comment:
How did we wind up with two types like this in the first place? WIll look for the original PR here...

Reply:
https://github.com/artsy/metaphysics/pull/3710#issuecomment-1359530966

Reply:
We understand that this means you can't re-use fragments between these two types right?

Reply:
Actually was a mess around the types, the new type added exposed properties we also want in a `Conversation`.
`Conversation.fromUser.collectorProfile` type is `CollectorProfileType` while from the `inquiry` was `InquirerCollectorProfile`. But yeah, also to re-use fragments but also to make the types consistent exposing the same stuff in different areas of MP.

Reply:
But it's the same underlying Gravity model, no? We don't need to mask fields depending on context. If it's a totally different model in Gravity, that's a different story.

Reply:
Yes, I said that to @lidimayra, fields like `isActiveBidder` and etc should be virtual fields in the respective model and come directly here, no masking needed.